README.md

```diff
@@ -1,2 +1,24 @@
-# Blog-app
-This is a blog app created with ruby, it displays all users and user's posts
+# Blog app with Rails
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
```

Comment:
- [ ] I would recommend updating the README file with title, description, author, and right steps to clone and run the project to make it professional.